src/llama-grammar.h

```diff
@@ -3,6 +3,7 @@
 #include "llama-impl.h"
 
 #include <map>
+#include <unordered_map>
```

Comment:
Should we use `unordered_map` vs `map`? Is there any benefit or necessity of a sorted key? @ggerganov

Reply:
I can't think of any need to have a sorted key for this -- feels like `unordered_map` would be my default way to go on this one. If you're curious, a good set of profiling runs to test both options wouldn't be a bad exercise.